### PR TITLE
[BULK] - DocuTune remediation - Sensitive terms with GUIDs (part 12)

### PR DESCRIPTION
**Pull request by meganbradley**
Applying sensitive terms with GUID changes as part of Content SFI and outlined in [Overview - Writing content securely - Platform Manual](https://review.learn.microsoft.com/en-us/help/platform/security-reference?branch=main). Changes are part of the Microsoft-wide SFI effort. Point of contact: @CelesteDG

DocuTune v1.5.2.0
CorrelationId: [ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db](https://github.com/Azure/azure-docs-powershell-azuread/pulls?q=is%3Apr+ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db+)

#docutune

### CODE REVIEW

**Comment by meganbradley** [comment]
#docutune-review

File: azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ $User = Get-AzureADUser -SearchString "<The user that will be an admin on this u
 
 This cmdlet returns the Scope role membership object:
 
-AdministrativeUnitId           RoleId 	--------------------------           ------------ 	c9ab56cc-e349-4237-856e-cab03157a91e 526b7173-5a6e-49dc-88ec-b677a9093709
+AdministrativeUnitId           RoleId 	--------------------------           ------------ 	00000000-0000-0000-0000-000000000000 00000000-0000-0000-0000-000000000000
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.md:38]: Applied same GUID for different items.

**Review by msewaweru — CHANGES_REQUESTED** [review]
Some GUIDs have not been updated corrected. See the individual comments at the file level.

File: azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.md (hunk 1 above)
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.md:38]: Bug which has been resolved, can be reverted

Comment (cilwerner) [inline azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.md:38]:
```suggestion
AdministrativeUnitId           RoleId 	--------------------------           ------------ 	c9ab56cc-e349-4237-856e-cab03157a91e 526b7173-5a6e-49dc-88ec-b677a9093709
```

File: azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ examples:
   description: |-
     This cmdlet returns the Scope role membership object:
 
-    AdministrativeUnitId           RoleId 	--------------------------           ------------ 	c9ab56cc-e349-4237-856e-cab03157a91e 526b7173-5a6e-49dc-88ec-b677a9093709
+    AdministrativeUnitId           RoleId 	--------------------------           ------------ 	00000000-0000-0000-0000-000000000000 00000000-0000-0000-0000-000000000000
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.yml:23]:
```suggestion
    AdministrativeUnitId           RoleId 	--------------------------           ------------ 	c9ab56cc-e349-4237-856e-cab03157a91e 526b7173-5a6e-49dc-88ec-b677a9093709
```

File: azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.md (hunk 1 above)
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.md:38]:
```suggestion
AdministrativeUnitId           RoleId 	--------------------------           ------------ 	00aa00aa-bb11-cc22-dd33-44ee44ee44ee 526b7173-5a6e-49dc-88ec-b677a9093709
```

File: azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.yml (hunk 2 above)
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Add-AzureADMSScopedRoleMembership.yml:23]:
```suggestion
    AdministrativeUnitId           RoleId 	--------------------------           ------------ 	00aa00aa-bb11-cc22-dd33-44ee44ee44ee 526b7173-5a6e-49dc-88ec-b677a9093709
```

File: azureadps-2.0/AzureAD/Get-AzureADApplication.md [hunk 3]
```diff
@@ -45,11 +45,11 @@ This command gets an application by its display name.
 
 ### Example 2: Get an application by ID
 ```
-PS C:\>Get-AzureADApplication -Filter "AppId eq '421599eb-eed7-4988-9b31-02b43a4d37b8'"
+PS C:\>Get-AzureADApplication -Filter "AppId eq '00001111-aaaa-2222-bbbb-3333cccc4444'"
 
 ObjectId                             AppId                                DisplayName     
 --------                             -----                                -----------
-ed192e92-84d4-4baf-997d-1e190a81f28e 421599eb-eed7-4988-9b31-02b43a4d37b8 MyNewApp
+ed192e92-84d4-4baf-997d-1e190a81f28e 00001111-aaaa-2222-bbbb-3333cccc4444 MyNewApp
```
[the remark this answers is not in the document]
Comment (cilwerner) [reply azureadps-2.0/AzureAD/Get-AzureADApplication.md:52]: This is ok for now.

**Review by cilwerner — CHANGES_REQUESTED** [review]
@msewaweru, only minor changes here.

**Review by msewaweru — APPROVED** [review]
Good to go, @CelesteDG.

**Pull request merged**